setup.cfg

```diff
@@ -64,10 +64,18 @@ setup_requires =
     setuptools_scm >= 1.15.0
     setuptools_scm_git_archive >= 1.0
 
-# These are required in actual runtime:
+# These are required in actual runtime, with few remarks
+# * ansi2html not really needed but enables ANSI coloring on pytest-html
+# * pytest-html not really needed but high-likely to be used
+#   https://github.com/pytest-dev/pytest-html/issues/282
+# These added as requirements because recent changes related to them broke
+# many projects using pytest-molecule. By listing them as dependencies we
+# avoid installing versions that are not compatible.
 install_requires =
-    molecule >= 3.0a9
+    ansi2html
+    molecule >= 3.0
     pytest >= 3.50
+    pytest-html<2.0
```

Comment:
SHouldn't this be <=2.0.1 ?

Reply:
Nope, there is slight chance that it may endup installing 2.0 on py27 which is not good. It could be better with extra conditions based on python version, but this can wait.

Reply:
Fair enough, looks good.